lib/api/dsl/match/matchNotExists.js

```diff
@@ -11,14 +32,13 @@ const strcmp = require('../util/stringCompare');
  * @param {object} document
  */
 function MatchNotExists (storage, testTables, document) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this 'MatchNotExists' function to match the regular expression ^[_a-z][a-zA-Z0-9]*$ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS100)